.github/workflows/build_iso.yml

```diff
@@ -39,24 +39,27 @@ jobs:
             image_tag: gts
     steps:
       - name: Build ISOs
-        uses: ublue-os/isogenerator@1.0.9
+        uses: jasonn3/build-container-installer@main
```

Comment:
I'd like to pin a version here instead of main. (We'll get our updates via dependabot)

Reply:
main is being used for testing for now until he publishes a release.

Reply:
Gotcha!